src/mol-plugin-state/formats/trajectory.ts

```diff
@@ -112,6 +116,24 @@ export const XyzProvider: TrajectoryFormatProvider = {
     visuals: defaultVisuals
 };
 
+export const LammpsDataProvider: TrajectoryFormatProvider = {
+    label: 'Lammps Data',
+    description: 'Lammps Data',
+    category: TrajectoryFormatCategory,
+    stringExtensions: ['data'],
+    parse: directTrajectory(StateTransforms.Model.TrajectoryFromLammpsData),
+    visuals: defaultVisuals
+};
+
+export const LammpsTrajDataProvider: TrajectoryFormatProvider = {
+    label: 'Lammps Traj Data',
+    description: 'Lammps Traj Data',
+    category: TrajectoryFormatCategory,
+    stringExtensions: ['lammpstrj'],
+    parse: directTrajectory(StateTransforms.Model.TrajectoryFromLammpsTrajData),
```

Comment:
Maybe use *LammpsTrajectory throughout. Would match  LammpsTrajectoryFile

Reply:
I try to rename and I ended up with : 
LammpsDataProvider: TrajectoryFormatProvider
LammpsTrajectoryDataProvider: TrajectoryFormatProvider
and 
LammpsTrajectoryProvider = DataFormatProvider
the .lammps should be both loadable directly and as a coordinate file for the .dat. What about maybe in trajectory.ts 
LammpsProvider: TrajectoryFormatProvider
LammpsTrajectoryProvider: TrajectoryFormatProvider
and in coordinate.ts
LammpsTrajectoryDataProvider: DataFormatProvider